pages/index.tsx

```diff
@@ -1,49 +1,153 @@
-import { NextPage } from 'next'
+import { NextPage, GetStaticProps } from 'next'
 import styled from 'styled-components'
-import { GetStaticProps } from 'next'
-import { ServicePreview, getServicePreviews } from '../cms/services'
 
-const Title = styled.h1`
-  font-size: 50px;
+import { Service, getServices } from '../cms/services'
+import { CardList } from '../src/Components/CardList'
+import { Layout } from '../src/Components/Layout'
+import { VisuallyHidden } from '../src/Components/VisuallyHidden'
+
+interface ListPageProps {
+  services: Array<ServicePreview>
+}
+
+export interface ServicePreview {
```

Comment:
This too looks like a local Picked interface? I couldn't work out why it was exported from here?